examples/hf-spaces-demo/README.md

```
@@ -52,10 +52,10 @@ This project is configured to **build directly on Hugging Face**. Just push the
 - **Build Command**: `npm run build`
 - **Output Path**: `dist/index.html`
 
-## 🔗 Links
+## Links
 
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)